docs/migration/file-native-writeback.md

```
@@ -0,0 +1,75 @@
+
+## Adapter migration table
+
+| Adapter | Canonical ID pattern | Example create draft path |
+|---|---|---|
+| Asana | `^\d+$` | `/asana/tasks/create request.json` |
+| ClickUp | `^[A-Za-z0-9]+$` | `/clickup/lists/<listId>/tasks/create request.json` |
+| GitHub | `^\d+$` | `/github/repos/<owner>/<repo>/pulls/<pullNumber>/reviews/create request.json` |
+| GitLab | `^[A-Za-z0-9_.:-]+$` | `/gitlab/projects/<namespace>/<project>/merge_requests/<iid>/discussions/create request.json` |
+| HubSpot | `^\d+$` | `/hubspot/contacts/create request.json` |
+| Intercom | `^[A-Za-z0-9_-]+$` | `/intercom/conversations/create request.json` |
+| Jira | comments: `^\d+$`; issues/projects: `^(?:[A-Z][A-Z0-9]+-\d+|\d+)$` | `/jira/issues/create request.json` |
```

markdownlint-cli2 (0.22.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "a-z0-9"

(MD052, reference-links-images)